hack/ansible-lint.sh

```diff
@@ -52,29 +49,25 @@ while getopts ":cpn" o; do
     esac
 done
 
+shift $((OPTIND-1))
+EXTRA_ARGS="$@"
+
+if ! which $BIN; then
+    echo "You need to install ansible-lint. Ideally from the source code branch"
+    echo "Example:"
+    echo -e "pip install 'ansible-lint[lock] @ git+https://github.com/ansible/ansible-lint@v6'"
+    exit 1
+fi
+
 # Be verbose from this point forward
 set -x
 
 cd "$TOPDIR"
 
-if [ "$CLEAN" == "yes" ]; then
-    rm -fv roles/*/.linted
-fi
-
-for role in roles/*; do
-    if [ "$CLEAR" == "yes" ]; then
-        clear
-    fi
-    echo "$role"
-    if [ -f "$role/.linted" ]; then
-        continue
-    else
-        if ansible-lint --profile="$PROFILE" "$role"; then
-            touch "$role/.linted"
-        else
-            break
-        fi
-    fi
-done
+ansible-lint \
```

Comment:
Right before this line could you do an `env` 
I got the feeling that we are seeing different behaviors because something in the environment